pandas/tests/plotting/test_frame.py

```diff
@@ -204,6 +204,24 @@ def test_color_and_style_arguments(self):
         with pytest.raises(ValueError):
             df.plot(color=["red", "black"], style=["k-", "r--"])
 
+    def test_color_and_marker(self):
+        # GH21003 - code sample from 2018-05-10 is equivalent to this test
+        df = DataFrame(np.random.random((7, 4)))
+        # combining a color string and a marker letter should be allowed
+        df.plot(color="green", style="d")  # d for diamond
+
+    def test_color_list_and_marker(self):
+        # GH21003 - code sample from 2020-04-23 is equivalent to this test
+        df = DataFrame(np.random.random((7, 4)))
+        # combining a color list and a marker letter should be allowed
+        color = ["yellow", "red", "green", "blue"]
+        ax = df.plot(color=color, style="d")
+        output_colors = [line.get_color() for line in ax.lines]
+        assert output_colors == color  # each "line" is one of the four colors
+        # Before this patch was introduced, the result was like this:
```

Comment:
move this comment to the to of the test as its confusing here

Reply:
I reworded this. The comment really fits best right after `df.plot` and not at the top.